app/src/composables/use-item.ts

```diff
@@ -278,6 +278,7 @@ export function useItem(collection: Ref<string>, primaryKey: Ref<string | number
 					params: {
 						fields: [relatedPrimaryKeyField!.field, ...fieldsToFetch],
 						[`filter[${relatedPrimaryKeyField!.field}][_in]`]: existingIds.join(','),
+						limit: -1
```

Comment:
I don't think adding `limit: -1` here is a scalable solution 😬 because this logic uses all existing IDs in a single query hitting the max length limit is a risk here.